.github/workflows/cmake.yml

```
@@ -688,6 +682,18 @@ jobs:
           Get-Command ccache
           (Get-Command ccache).Source
           ccache --show-stats --verbose --version
+      - name: Download Apache Arrow latest source archive and verify via PGP
+        run: |
+          $LATEST_RELEASE_INFO = Invoke-RestMethod "https://api.github.com/repos/apache/arrow/releases/latest"
+          # e.g. "apache-arrow-19.0.0"
+          $APACHE_ARROW_VERSION = $LATEST_RELEASE_INFO.tag_name
+          # "apache-arrow-19.0.0" -> "19.0.0"
+          $ARROW_VERSION = $APACHE_ARROW_VERSION -replace '^apache-', ''
+          $DOWNLOAD_URL = "https://dlcdn.apache.org/arrow/${ARROW_VERSION}/${APACHE_ARROW_VERSION}.tar.gz"
```

We should use `closer.lua` URL for ASF artifacts:  https://infra.apache.org/release-download-pages.html#download-page

```suggestion
          $DOWNLOAD_URL = "https://www.apache.org/dyn/closer.lua/arrow/${ARROW_VERSION}/${APACHE_ARROW_VERSION}.tar.gz?action=download"
```

---

fix: [c81b06e](https://github.com/groonga/groonga/pull/2209/commits/c81b06e44979a95be0fdae87f6214abf74d02d70) Thanks.